econml/dml.py

```diff
@@ -595,5 +665,5 @@ def transform(self, X):
         super().__init__(model_y=model_y, model_t=model_t,
                          model_final=ElasticNetCV(),
                          featurizer=RandomFeatures(random_state),
-                         fit_cate_intercept=False,
+                         fit_cate_intercept=fit_cate_intercept,
```

Comment:
[minor, mostly out of curiosity]
Shouldn't the random features already approximate a constant well so that an intercept is unnecessary?

Reply:
Yeah but having an intercept always good for regularization. Also it breaks the X=None case and this was an easy fix.